contrib/hadoop-store-builder/src/java/voldemort/store/readonly/mr/azkaban/VoldemortBuildAndPushJob.java

```diff
@@ -212,10 +214,18 @@ public VoldemortBuildAndPushJob(String name, azkaban.utils.Props azkabanProps) {
                 if (clusterURLs.contains(url)) {
                     throw new VoldemortException("the URL: " + url + " is repeated in the "+ PUSH_CLUSTER + " property ");
                 }
-                this.clusterURLs.add(url);
-                AdminClient adminClient = createAdminClient(url, fetchAllStoresXml);
-                this.adminClientPerCluster.put(url, adminClient);
-                this.closeables.add(adminClient);
+                try {
+                    AdminClient adminClient = createAdminClient(url, fetchAllStoresXml);
+                    this.clusterURLs.add(url);
+                    this.adminClientPerCluster.put(url, adminClient);
+                    this.closeables.add(adminClient);
+                } catch (Exception e) {
```

Comment:
Yes.
From user's perspective, successful job means that the data is ready in every colo, and it can be easily inferred through job status.
Otherwise, it might too hard for them to check the job logs every time.
Also it might not be easy for us to track job status since normally we only check whether the job fails or not.